R/downloadableTable.R

```diff
@@ -118,7 +120,7 @@ downloadableTableUI <- function(id,
 #' server.R using the same id provided in \code{downloadableTableUI}:
 #'
 #' \strong{\code{callModule(downloadableTable, id, logger, filenameroot,
-#' downloaddatafxns, tabledata, rownames, caption)}}
+#' downloaddatafxns, tabledata, selection, rownames, caption)}}
```

Comment:
need to move this to the end of the function for backwards compatibility for users who do not use named inputs.